app/controllers/api/teams.php

```diff
@@ -356,6 +385,7 @@
                         Permission::delete(Role::user($userId)),
                     ],
                     'email' => $email,
+                    'phone' => $phone,
```

Comment:
Above this we define `$userId = ID::unique();`. Should we take it from parameters instead, if provided?

Reply:
No, if the ID is provided and user doesn't exist with that ID we throw 404 don't create new with the given ID